src/Aggregation/Reduced.php

```diff
@@ -42,7 +46,7 @@ public function __construct(Arrayee $arrayee, callable $callback, Arrayee $initi
     /**
      * @param Arrayee $arrayee
      * @param callable $callback Function that reduces items to array <code>function(array $carry, $item): array</code>
-     * @return Reduced
+     * @return Reduced<V>
```

Comment:
is it somehow supported by phpstorm?

Reply:
yep, below you can see a warning if I remove a template definition from Reduced class
![image](https://user-images.githubusercontent.com/606705/103676009-a3175680-4f91-11eb-9a74-e136cdf09444.png)

Also, [here](https://blog.jetbrains.com/phpstorm/2020/12/phpstorm-2020-3-release/) described more detailed (see Psalm and PHPStan Support section)